prep-html-skeleton/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE >
+<html>
+  <head>
+    <title>HTML Skeleton Exercise</title>
+    <body>
+      <h1>I built a proper HTML skeleton!</h1>
+    </body>
+  </head>
```

Comment:
Hi Kevin,

For your HTML skeleton, you'll want to separate your `<head>` and `<body>` so that they're next to each other instead of within each other.

```html
  <head>
    <title>HTML Skeleton Exercise</title>
  </head>
  <body>
    <h1>I built a proper HTML skeleton!</h1>
  </body>
```